src/test/resources/project-alt1-stub/pom.xml

```diff
@@ -56,15 +56,15 @@
 					<releaseBranchProperties>
 						<foo>bar</foo>
 					</releaseBranchProperties>
-					<scope>system</scope>
+					<otherDeployBranchPattern>(origin/)feature/poc/.*</otherDeployBranchPattern>
```

Comment:
Did you intend for the `(origin/)` group to be optional?  If so, this is missing a `?`.

Reply:
Yup.